src/perf/lib/PerfServer.cpp

```diff
@@ -284,6 +404,25 @@ PerfServer::SendResponse(
     }
 }
 
+void
+PerfServer::SendDelayedResponse(
+    _In_ StreamContext* Context,
+    _In_ void* Handle,
+    _In_ bool IsTcp
+    )
+{
+    uint16_t WorkerNumber = (uint16_t)CxPlatProcCurrentNumber();
+    CXPLAT_DBG_ASSERT(WorkerNumber < ProcCount);
+    Context->AddRef();
+    if (IsTcp) {
+        //
+        // TcpConnection object is separately reference counted
+        //
+        CXPLAT_DBG_ASSERT(((TcpConnection*)Handle)->TryAddRef());
```

Comment:
I don't think DBG assert actually executes the inner logic outside of debug builds.